src/core/ext/filters/client_channel/subchannel.h

```diff
@@ -68,6 +68,8 @@
 
 namespace grpc_core {
 
+extern DebugOnlyTraceFlag grpc_trace_subchannel;
```

Comment:
The flag is only used in `subchannel.cc`, we don't need to declare it here.

Also, the `grpc_trace_stream_refcount` doesn't need to be declared in the header any more. 

Looks like some pre-existing tracers will be declared in the headers even though only the according .cc files are using them. Anyways, I don't think it's necessary.